packages/flutter_tts/tizen/src/text_to_speech.cc

```diff
@@ -245,6 +273,14 @@ bool TextToSpeech::GetSpeedRange(int32_t *min, int32_t *normal, int32_t *max) {
   return true;
 }
 
+bool TextToSpeech::IsLanguageAvailable(const std::string &language) {
+  if (std::find(supported_lanaguages_.begin(), supported_lanaguages_.end(),
```

Comment:
How about simply immediate return?
`return std::find(supported_lanaguages_.begin(), supported_lanaguages_.end(),language) != ported_lanaguages_.end();`

Reply:
Thank you for your review.